man/systemd.service.xml

```diff
@@ -563,6 +563,31 @@
         </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>TimeoutAbortSec=</varname></term>
+        <listitem><para>This option configures the time to wait for the service to terminate when it was aborted due to a
+        watchdog timeout (see <varname>WatchdogSec=</varname>). If the service has a short <varname>TimeoutStopSec=</varname>
+        this option can be used to give the system more time to write a core dump of the service. Upon expiration the service
+        will be forcibly terminated by <constant>SIGKILL</constant> (see <varname>KillMode=</varname> in
+        <citerefentry><refentrytitle>systemd.kill</refentrytitle><manvolnum>5</manvolnum></citerefentry>).  </para>
+
+        <para>Takes a unit-less value in seconds, or a time span value such as "5min 20s". Pass <literal>0</literal> to skip
+        the dedicated watchdog abort timeout handling and fall back <varname>TimeoutStopSec=</varname>. Pass
+        <literal>infinity</literal> to disable the timeout logic. Defaults to <varname>DefaultTimeoutAbortSec=</varname> from
+        the manager configuration file (see
+        <citerefentry><refentrytitle>systemd-system.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>) which itself
+        defaults to <literal>0</literal>.
+        </para>
+
+        <para>If a service of <varname>Type=notify</varname> sends <literal>EXTEND_TIMEOUT_USEC=…</literal>, this may cause
+        the abort time to be extended beyond <varname>TimeoutAbortSec=</varname>. The first receipt of this message
```

Comment:
hmm, this is a bit weird, no? the service can hardly send a msg like that if has segfaulted and being dumped ...

Reply:
The service could catch SIGABRT. Embedded Java VMs are one example that do some special kind of core-dumping if they are configured so.

It's also how the code works at the moment anyway. I have no problem to remove it from the documentation, though. But I would not change the code to prevent this feature because it would make the code more complicated for no good reason.

Reply:
ok